src/utils/reset-css.js

```diff
@@ -88,12 +88,12 @@ const resetCss = css`
   audio,
   video {
     box-sizing: border-box;
-    all: initial;
```

Comment:
why did you remove this? 
the reset-css should try to reset the global styles all custom styles should then be applied specifically to the widget.